source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,101 @@
+.. _csharp-update-many:
+
+=====================
+Update Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can update more than one document at once using the `UpdateMany() <{+api-root+}/M_MongoDB_Driver_IMongoCollection_1_UpdateMany.htm>`__ method on
```

Comment:
Question (possible issue): I thought the convention was to only provide inline links at the end of the page/section? Is there a reason inline links are provided for `UpdateMany`, `UpdateResult`, and `UpdateOptions`?

Reply:
Do you have a link for that convention? I was using [this page](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/) and didn't see anything there.

I included the links to immediately show the reader where to go for more info.

Reply:
Unfortunately I don't have a link, it's not in the style guide (just a convention arrived at in a team meeting as we all had different linking styles). I think this convention is used in [go usage examples](https://www.mongodb.com/docs/drivers/go/current/usage-examples/findOne/).

Reply:
The Go page has some inline links; is there a certain kind we're trying to avoid?

Reply:
Yes,  generally we try to put inline links at the end of sections. We do this so as to not give people a fragmented experience while reading something (we felt that readers shouldn't feel the need to disrupt flow of passage to go somewhere else in the main paragraph if it can be avoided. Sometimes it makes sense to add an inline line mid paragraph but often they can be removed.).

So we try to avoid inline links in the beginning or middle of paragraphs. try to save inline links for the end of a section if possible. In this case, it may be best to save the ``UpdateMany()`` link for the API documentation section as going to that page at the beginning may not help them understand the usage example (which is designed to be legible and self contained).

Reply:
Thanks--makes sense. I'll take them out.